### ISSUE TITLE
Vanilla command blocks are unsupported.

### ISSUE THREAD

**Issue by Frustrated-Programmer** [MightyPirates/Scannable#155]
I'm working on a very *large* scale project, and I wanted to find all the command blocks in our project. This mod seems perfect for that, however the scanner seems incompatible with command blocks, it perfectly works with other blocks I tested, but couldn't find command blocks. 

1.20.1 - Fabric

**Edit:** Chain and Repeating command blocks *can* and are found, but the standard regular command blocks don't work. No idea why.

**Comment by Frustrated-Programmer** [comment]
I found the cause, I think.
https://github.com/MightyPirates/Scannable/blob/1.20.4/common/src/main/java/li/cil/scannable/common/config/CommonConfig.java#L93-L100

Turns out you are intentionally ignoring command blocks for some reason. The comment suggests this is because of ores, but I can't imagine players ever getting a command block or being able to detect command blocks in any real capacity. 

Would love it if this is read from maybe a config.json or something else. 
I'd love to remove this line and rebuild the mod for 1.20.1, but I have no idea how todo that or how to start and I'd appreciate it if this is updated to be removed.
